src/ontology/genepio-idranges.owl

```diff
@@ -1,6 +1,6 @@
 Prefix: rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 Prefix: idsfor: <http://purl.obolibrary.org/obo/IAO_0000598>
-Prefix: dc: <http://purl.org/dc/elements/1.1/>
+Prefix: dc: <http://purl.org/dc/terms/>
```

Comment:
OBO foundry has started (or has been in some cases) been using a slightly longer dcterms: prefix for http://purl.org/dc/terms/

Prefix(dcterms:=<http://purl.org/dc/terms/>)

Reply:
Resolved commit 64be85a775fb3be6611e4dc9e1ef4d89c1123f94